lookout/core/lib.py

```diff
@@ -218,7 +218,7 @@ def parse_files(filepaths: Sequence[str], content_getter: callable, line_length_
     return size_passed
 
 
-def filter_files(files: Sequence[File], line_length_limit: int, overall_size_limit: int,
+def filter_files(files: Dict[str, File], line_length_limit: int, overall_size_limit: int,
```

Comment:
This will break `style-analyzer` because it expects a sequence. Please leave it untouched.

Reply:
no, vice versa. that is the point of this PR.
I need to modify this to leave style-analyzer FormatAnalyzer logic untouched.

Reply:
Or we can modify `files_by_language` to return not a `Dict[str, Dict[str, File]]` but `Dict[str, List[File]]`

Reply:
🤔

Reply:
Indeed. Weird, I copy-pasted that